codegenerator/cli/src/config_parsing/human_config.rs

```diff
@@ -537,6 +537,11 @@ pub mod fuel {
             description = "Index Mint receipts. The option can be omitted when the event name is Mint."
         )]
         pub mint: Option<bool>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        #[schemars(
+            description = "Index Burn receipts. The option can be omitted when the event name is Burn."
+        )]
+        pub burn: Option<bool>,
```

Comment:
When would we want to use this flag? It looks like you just add "Mint" or "Burn" to the name which seems like a good idea. Also looks like you can't have it with a log_id with mint or burn.

Is it so that you can alias it with a different name if you want?

Reply:
Yes, this is for aliasing